pkg/build/buildutil/buildutil.go

```diff
@@ -83,3 +94,15 @@ func ConfigNameForBuild(build *buildv1.Build) string {
 	}
 	return build.Labels[buildv1.BuildConfigLabelDeprecated]
 }
+
+// NameForBuildVolume returns a valid pod volume name for the provided build volume name.
+func NameForBuildVolume(objName string) string {
+	// Volume names must be a valid DNS Label - see https://kubernetes.io/docs/concepts/overview/working-with-objects/names/#dns-label-names
+	return naming.GetName(objName, BuildVolumeSuffix, kvalidation.DNS1123LabelMaxLength)
```

Comment:
Since we know we need this to be lowercase, perhaps we need a strings.ToLower call as well?

Reply:
updated in current push.